### PR TITLE
add gdiScaling support

### PR DESCRIPTION
**Pull request by fredericosilva**
Hello,

I need that support for my app, decided to give back to the community. 

Let me know if you need any clarification

Thank you for putting this project together,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,13 @@ Appends `dpiAware` tag for custom DPI aware applications.
       on_off         (optional) true/false or 0/1. Default is 0
       per_monitor    (optional) win81 per-monitor DPI awareness. Default is 0
 
+#### GdiScaling
+
+Appends `gdiScaling` tag for enabling GDI scaling
+
+    Parameters       [on_off] [per_monitor]
```
Comment (wqweto) [inline README.md]: Is `[per_monitor]` redundant here?

Comment (fredericosilva) [reply README.md]: It's not, good old fashion copy and past issue.

**Comment by wqweto** [comment]
Merged in 3616d22e0988726f24a01aeeab5e87573bb7a9ec

**Comment by wqweto** [comment]
https://github.com/wqweto/UMMM/releases/tag/1.0.13

**Pull request closed without merging**